docs/quickstart_guide.md

```diff
@@ -240,14 +240,16 @@ scripts/cloudlab/start_onenode_vhive_cluster.sh
 2. Run the deployer client:
```

Comment:
in 1., specify that the quickstart guide describes only synchronous functions, and multi-function apps, deployment and benchmarking. Refer to the benchmarking guide for other cases, e.g., async workflows.